pom.xml

```diff
@@ -38,6 +39,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>${maven-resource-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>verify</phase>
```

Comment:
When I executed `mvn package` the charts directory stayed empty. (I made sure to remove the `SNAPSHOT` exclusion for this test). Probably the phase should be `package`

Reply:
DONE